### PR TITLE
cdc/sink: kafka sink integrate broker level metrics

### LINKED ISSUE #4588: add sarama broker level metrics

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### What problem does this PR solve?

Issue Number: close #4588

### What is changed and how it works?

add broker level metrics

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Screen Shot 2022-02-14 at 3 21 31 PM

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch

### Release note 

```release-note
`None`.
```

### CODE REVIEW

File: cdc/sink/producer/kafka/kafka.go [hunk 1]
```diff
@@ -262,6 +263,15 @@ func (k *kafkaSaramaProducer) Close() error {
 			zap.String("changefeed", k.id), zap.Any("role", k.role))
 	}
 
+	start = time.Now()
+	if err := k.admin.Close(); err != nil {
+		log.Warn("close kafka cluster admin with error", zap.Error(err),
```
Comment (asddongmen) [inline cdc/sink/producer/kafka/kafka.go:268]: should also add duration in err log.

**Review by asddongmen — APPROVED** [review]

**Review by overvenus — APPROVED** [review]

**Pull request merged**